client/me/privacy/main.jsx

```diff
@@ -135,9 +156,73 @@ const Privacy = createReactClass( {
 						</FormButton>
 					</form>
 				</Card>
+				<SectionHeader label={ translate( 'Legal' ) } />
+				<Card>
+					<p>
+						{ translate(
+							'A Data Processing Addendum (DPA) allows web sites and companies to assure customers, vendors, ' +
+								'and partners that their data handling complies with the law.'
+						) }
+					</p>
+
+					<p>{ translate( 'Note: most free site owners or hobbyists do not need a DPA.' ) }</p>
+
+					<p>
+						<strong>
+							{ translate(
+								'Having a DPA does not change any of our privacy and security practices for site visitors. ' +
+									'Everyone using our service gets the same high standards of privacy and security.'
+							) }
+						</strong>
+					</p>
+					<Button primary className="privacy__dpa-request-button" onClick={ this.props.requestDpa }>
+						{ translate( 'Request a DPA' ) }
+					</Button>
+				</Card>
 			</Main>
 		);
 	},
 } );
 
-export default compose( localize, protectForm )( Privacy );
+const dpaRequestId = 'dpa-request';
+function requestDpa() {
+	requestHttpData(
+		dpaRequestId,
+		http( {
+			apiNamespace: 'wpcom/v2',
+			method: 'POST',
+			path: '/me/request-dpa',
+		} ),
+		{
+			freshness: -Infinity,
```

Comment:
Wanting to confirm here that you mean "never ever cull this call because I want to always certainly make the actual HTTP request even if it's only been 0.000001ms since I last made it"

I see several examples of `-Infinity` in the codebase and they are probably all meant to be `Infinity` which means "no matter hold old the data is it's fine, don't make the call again"

Reply:
Yeah, I mean that they can request as often as they want. But we should probably disable the button and change the label while the request is pending so they can't issue simultaneous requests, assuming http-data doesn't already deny them when there is an outstanding request with the same ID

Reply:
I'll add a "why" comment to that